Library/ViewModels/RewardCellViewModel.swift

```diff
@@ -372,6 +373,14 @@ private func footerString(project: Project, reward: Reward) -> String {
     .joined(separator: " • ")
 }
 
+private func shouldShowDecimal(_ backing: Backing) -> String {
```

Comment:
Functions and variables starting with `should` usually return / are of type `Bool`. TableView, for example, has methods like `shouldHighlightRowAt` and `shouldShowMenuForRowAt`. I would rename this function to a name that helps to understand what it's returning exactly. Something like `backingAmount(for backing: Backing)`